src/Microsoft.AspNet.Mvc.Extensions/Filters/ValidateAntiforgeryTokenAttribute.cs

```diff
@@ -2,19 +2,20 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNet.Antiforgery;
 using Microsoft.Framework.DependencyInjection;
 
 namespace Microsoft.AspNet.Mvc
 {
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
-    public class ValidateAntiForgeryTokenAttribute : Attribute, IFilterFactory, IOrderedFilter
+    public class ValidateAntiforgeryTokenAttribute : Attribute, IFilterFactory, IOrderedFilter
```

Comment:
Note this is a compat break from MVC 5. Attribute is named `ValidateAntiForgeryTokenAttribute` there.